src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,13 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
```

Comment:
It will be better if we render this component only when the user exists

```suggestion
  user: User;
```